src/index.js

```diff
@@ -245,6 +272,7 @@ class HCaptcha extends React.Component {
     handleExpire () {
       const { onExpire } = this.props;
       const { captchaId } = this.state;
+      const hcaptcha = this._hcaptcha;
```

Comment:
why not to use `this._hcaptcha` directly?
`this._hcaptcha.reset(captchaId);`

Reply:
It's not necessary to remove the word `this` from variable names, but it depends on the coding conventions and style guide that we want to follow.

I like removing the `this` from the `hcaptcha` variable name and so on, it makes the code a bit more readable - but I think this a personal preference only.

Since this is a "public react library", it should be clean and easy for developers to interprete.

Reply:
Less legacy code to update, was my thought